core/modules/admin_bar/admin_bar.module

```diff
@@ -237,6 +237,9 @@ function admin_bar_preprocess_page(&$variables) {
   }
 
   $config = config('admin_bar.settings');
+  if ($setting = $config->get('local_actions_bottom')) {
+    $settings['local_actions_bottom'] = $setting;
+  }
```

Comment:
Do we really need to send this setting to JS? It's configured in `core/modules/admin_bar/admin_bar.inc` above, so I think we can skip this...

Reply:
You might be right about this. I was going for consistency (send this to JS, like we do with the other 2 settings), but unlike these other settings, this setting has nothing to do with the JS placement of the admin bar (which is what the JS is doing) - only with the weight that's being assigned to the menu items.

I'll remove it 👍🏼

Reply:
Done!